lib/pages/news_feed/view/news_feed_page.dart

```diff
@@ -24,6 +24,23 @@ class NewsFeedPage extends StatelessWidget {
           }
 
           final List<NewsFeedItem> newsFeedItems = snapshot.data;
+          if (newsFeedItems == null) {
+            return Center(
+                child: Padding(
+                    padding: const EdgeInsets.all(8),
+                    child: Text(
+                      S.of(context).warningUnableToReachNewsFeed,
+                      textAlign: TextAlign.center,
+                      style: TextStyle(color: Theme.of(context).disabledColor),
+                    )));

```

Comment:
Nit: Add trailing commas after the last parameter when the constructor doesn't fit on one line. It gets formatted nicer (each closed paran will be on its own line, it's easier to read and follow the nested structure).